client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -384,8 +383,34 @@ export default {
       deletePage: 'page/delete',
       updatePageSet: 'page/updateSet',
       createPage: 'page/create',
+      loadPages: 'page/load',
     }),
 
+    // helper to count pages with the same name
+    countDuplicatePages (pageTitles) {
```

Comment:
Are we counting duplicate pages or duplicate page titles? Also the param is called pageTitles, no need to map it before passing it in, just pass the pages. Since we were talking about faster/slower, well you do a for loop here (map) for no reason so that is another O(n) slower